packages/manager/apps/web/client/app/hosting/database/order-private/translations/Messages_fr_FR.json

```
@@ -0,0 +1,47 @@
+  "hosting_database_order_private_description": "Vous devez avoir un hébergement pour commander les options SQL Privé et Start SQL.",
+  "hosting_database_order_private_step_header_type": "Type de base de données",
+  "hosting_database_order_private_step_header_customize": "Personnalisation",
+  "hosting_database_order_private_step_header_duration": "Choisissez la durée",
+  "hosting_database_order_private_step_header_activation": "Activation",
+  "hosting_database_order_private_step_header_payment": "Paiement",
+  "hosting_database_order_private_step_description_activation": "Afin de confirmer votre demande, veuillez accepter les conditions d'utilisation.",
+  "hosting_database_order_private_step_description_payment": "Afin de confirmer votre demande, veuillez choisir votre moyen de paiement, et accepter les conditions d'utilisation.",
+  "hosting_database_order_private_submit_activate": "Activer",
+  "hosting_database_order_private_submit_pay": "Payer",
+  "hosting_database_order_private_cancel": "Annuler",
+  "hosting_database_order_private_select_placeholder": "Sélectionner...",
+  "hosting_database_order_private_type": "Quelle offre de base de données désirez vous ?",
+  "hosting_database_order_private_type_label": "SQL Privé",
+  "hosting_database_order_private_type_text": "Bases de données aux performances mémoires (RAM) garanties pour votre hébergement Web Hosting OVH",
+  "hosting_database_order_private_domains_pack": "Sélectionnez un pack de base de données",
+  "hosting_database_order_private_domains_pack_help": "Attention, les packs de bases de données de mêmes tailles ne se cumulent pas. Si vous disposez déjà d’un pack SQL de même taille pour ce domaine, le nouveau pack remplacera l’ancien.",
+  "hosting_database_order_private_domains_pack_value_SQLPERSO_1_BASES_400_MB": "1 base de 400 Mo",
+  "hosting_database_order_private_domains_pack_value_SQLPERSO_1_BASES_800_MB": "1 base de 800 Mo",
+  "hosting_database_order_private_domains_pack_value_SQLPERSO_20_BASES_100_MB": "20 bases de 100 Mo",
+  "hosting_database_order_private_domains_pack_value_SQLPERSO_20_BASES_200_MB": "20 bases de 200 Mo",
+  "hosting_database_order_private_domains_pack_value_SQLPERSO_2_BASES_400_MB": "2 bases de 400 Mo",
+  "hosting_database_order_private_domains_pack_value_SQLPERSO_2_BASES_800_MB": "2 bases de 800 Mo",
+  "hosting_database_order_private_domains_pack_value_SQLPERSO_50_BASES_100_MB": "50 bases de 100 Mo",
+  "hosting_database_order_private_domains_pack_value_SQLPERSO_50_BASES_200_MB": "50 bases de 200 Mo",
+  "hosting_database_order_private_domains_pack_value_SQLPERSO_5_BASES_100_MB": "5 bases de 100 Mo",
+  "hosting_database_order_private_domains_pack_value_SQLPERSO_5_BASES_200_MB": "5 bases de 200 Mo",
+  "hosting_database_order_private_domains_pack_value_SQLPERSO_5_BASES_400_MB": "5 bases de 400 Mo",
+  "hosting_database_order_private_domains_pack_value_SQLPERSO_5_BASES_800_MB": "5 bases de 800 Mo",
```

This looks like it could be generalized 
```suggestion
  "hosting_database_order_private_domains_pack_value_SQLPERSO_5_BASES_800_MB": "{{ n }} bases de {{ storage }}",
```

---

It seems not used in fact... I'll will delete them